app/main.py

```diff
@@ -1 +1,47 @@
-# write your code here
+class Animal:
+
+    alive = []
+
+    def __init__(self,
+                 name: str,
+                 health: int = 100,
+                 hidden: bool = False) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        Animal.alive.append(self)
+        # Animal.alive.append(self.__dict__)
+        print(Animal.alive, len(Animal.alive))
+
+    # def __str__(self):
+    #     res = []
+    #     for _ in self.alive:
+    #         res.append(self.__dict__)
+    #     Animal.alive = res
+    #     return Animal.alive
```

Comment:
Add `__repr__` method to class `Animal` for better formatting
https://github.com/mate-academy/py-herbivores-and-carnivores/tree/master#:~:text=Also%20implement%20feature%20that%20when%20you%20print%20Animal.alive%20it%20should%20look%20like%20this%3A

Reply:
ok